DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -274,7 +274,8 @@ private Task RedirectToLogin(RedirectContext<CookieAuthenticationOptions> contex
         {
             var applicationPath = new Uri(config["AppRootPath"]).AbsolutePath.TrimEnd('/');
             var url = HttpUtility.UrlEncode(applicationPath + context.Request.Path);
-            context.HttpContext.Response.Redirect(config["AppRootPath"] + $"/Login?returnUrl={url}");
+            var queryString = HttpUtility.UrlEncode(context.Request.QueryString.Value);
+            context.HttpContext.Response.Redirect(config["AppRootPath"] + $"/Login?returnUrl={url}&queryString={queryString}");

```

Comment:
Could you change these two config calls to use our recent helper method, `config.GetAppRootPath()`?